src/plugins/fileextractor/fileextractor.cpp

```diff
@@ -216,6 +220,9 @@ event_response_t fileextractor::createfile_ret_cb(drakvuf_t,
             addr_t file = 0;
             auto filename = get_file_name(vmi, info, handle, &file, nullptr);
 
+            if (is_in_exclude_list(filename))
+                return VMI_EVENT_RESPONSE_NONE;
```

Comment:
shouldn't you call `createfile_ret_hooks.erase(hook_id);` here?

Reply:
You are right. Thanks!

Reply:
Fixed